internal/pkg/config/v1/config_test.go

```diff
@@ -760,17 +782,6 @@ func TestConfig_FindContext(t *testing.T) {
 	}
 }
 
-func TestConfig_DeleteContext(t *testing.T) {
-	cfg := &Config{
-		BaseConfig:     config.NewBaseConfig(new(version.Version)),
-		Contexts:       map[string]*Context{contextName: {Name: contextName}},
-		CurrentContext: contextName,
-	}
-
-	err := cfg.DeleteContext(contextName)
-	require.NoError(t, err)
-}
```

Comment:
When running this test alone, the whole config.json will be removed and won't be able to be reverted. This messed up the config as well, causing `unable to parse version from config` error.